packages/backend/pyproject.toml

```diff
@@ -13,21 +13,21 @@ authors = [
 ]
 requires-python = ">=3.10"
 dependencies = [
-  "click >= 7.0.0",
-  "pandas >= 2.2.0",
+  "click >= 8.0.0",
+  "pandas >= 2.3.0",
   "fastparquet >= 2024.0.0",
   "platformdirs >= 4.3.0",
   "umap-learn >= 0.5.0",
-  "sentence-transformers >= 3.3.0",
-  "fastapi >= 0.115.0",
-  "uvicorn >= 0.32.0",
+  "sentence-transformers >= 5.0.0",
+  "fastapi >= 0.116.0",
+  "uvicorn >= 0.35.0",
   "uvloop >= 0.21.0",
-  "pyarrow >= 18.0.0",
-  "duckdb >= 1.1.0",
+  "pyarrow >= 21.0.0",
+  "duckdb >= 1.3.2",
   "inquirer >= 3.0.0",
-  "llvmlite >= 0.43.0",
-  "accelerate >= 1.5.0",
-  "tqdm >= 4.60.0",
+  "llvmlite >= 0.44.0",
+  "accelerate >= 1.9.0",
+  "tqdm >= 4.67.0",
```

Comment:
I think for the Python package we'd want more generous versions requirements to make it easy to install our package together with other packages. Tighter versions requirements may lead to more conflicts?

Reply:
Good point. I'll just update the lockfile then.